app/docker/components/datatables/containers-datatable/containersDatatable.html

```diff
@@ -99,7 +99,7 @@
           </span>
         </div>
       </div>
-      <containers-datatable-actions
+      <containers-datatable-actions ng-if="$ctrl.state.endpointStatus === 1"
```

Comment:
I think that any part of a component that should be hidden related to the status of an endpoint should be referring to a binding such as `ng-if="!$ctrl.offlineMode"` instead of relying on the `endpointStatus` being retrieved inside each component (via `EndpointProvider`, see my comment below for more details).